recipes/osgearth/all/conanfile.py

```diff
@@ -1,9 +1,11 @@
-from conans import ConanFile, CMake, tools
-from conan.tools.files import rename
-from conans.errors import ConanInvalidConfiguration
-from conans.tools import os_info
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain
+from conan.tools.files import apply_conandata_patches, copy, get, rename, rmdir
+from conan.tools.microsoft import is_msvc
+from conan.tools.scm import Version
 import os
-import functools
 
 required_conan_version = ">=1.33.0"
```

Comment:
This is almost certainly wrong. However, I am not sure what to put here. The [Consuming Recipes documentation](https://github.com/conan-io/conan-center-index/blob/master/docs/consuming_recipes.md) suggests that we should be supporting Conan 2.0 (or, to be more precise, Conan 2.x, which I interpret as meaning "at least Conan 2.0"). Yet, I have seen several recipes in this repository whose `required_conan_version` value claims to still support some version of Conan 1.